src/utils.ts

```diff
@@ -104,11 +104,17 @@ export function getEnv(prefix = ''): (name: string) => string | undefined {
     envsLowerCase[envKey.toLowerCase()] = process.env[envKey];
   }
 
-  return (name: string) => {
+  return (name: string): string | undefined => {
     return (
       envsLowerCase[`${prefix}${name}`.toLowerCase()] ||
       envsLowerCase[name.toLowerCase()] ||
       undefined
     );
   };
 }
+
+export function setEnv(key: string, value: string | undefined): void {
```

Comment:
Nit: I think this can be
```suggestion
export function setEnv(key: string, value?: string): void {
```